views/js/runner/provider/qti.js

```diff
@@ -273,10 +299,7 @@ define([
 
                 submit()
                  .then(updateStats)
-                 .then(computeNextMove)
-                 .catch(function(err){
-                    self.trigger('error', err);
-                 });
+                 .then(computeNextMove);
```

Comment:
Did you try ? if the promise is returned, then only the first catch is called (it should be in the same promise chain), see http://jsbin.com/lusizur/edit?js,console